src/pages/Route/service.ts

```diff
@@ -50,6 +50,8 @@ export const fetchList = ({ current = 1, pageSize = 10 }, search: string) => {
 };
 
 export const remove = (rid: number) => request(`/routes/${rid}`, { method: 'DELETE' });
+export const offline = (rid: number) => request(`/offlineroutes/${rid}`, { method: 'PUT' });
+export const publish = (rid: number) => request(`/publishroutes/${rid}`, { method: 'PUT' });
```

Comment:
Not a good Pathname for this module,  I would prefer using `PUT /routes/${rid}/offline` if possible.

Reply:
done